articles/sso/current/single-page-apps.md

```diff
@@ -151,21 +171,21 @@ $(function () {
 });
 ```
 
-The response for a call made without the `prompt=none` parameter is indistinguishable from the response for a call made with the parameter.
+The response for a call made without the **prompt=none** parameter is indistinguishable from the response for a call made with the parameter.
 
 You can test this using the sample app. If you're already logged in, you can request an updated token by clicking the **Click here to renew it** link.
 
 At this point, the app silently authenticates you, gets the new token, and updates the page to reflect your new token expiration datetime. Notice that you did not see the Lock screen asking for your credentials.
 
-## Further Actions
+## Further actions
 
 If your authentication flow triggers an error (indicating unsuccessful authentication) at any point, you'll need to [handle the error(s)](/api-auth/tutorials/silent-authentication#refresh-expired-tokens) before moving on.
 
 If your Access Tokens expire, you can use [Silent Authentication](/api-auth/tutorials/silent-authentication#refresh-expired-tokens) to get a new one.
 
-## Single Logout
+## Single logout
 
-If the user logs out of `app1.com`, then you'll want the user's tokens cleaned up on `app2.com` and `app3.com`.
+If the user logs out of **app1.com**, then you'll want the user's tokens cleaned up on **app2.com** and **app3.com**.
 
 To implement [Single Log Out](/logout), you need to check periodically to see if Auth0 has expired the SSO session. If so, remove the token from the application's local storage to ensure that the local session clears.
```

Comment:
There are several places where it's `Single Log Out` vs `Single Logout` - Probably should make it consistent. According to our grammar rules, it'd be `Single Logout` and `Single Signin` since it's a noun and not verb, but that doesn't really mirror the acronym "SSO" and "SLO" very well, so...  kind of unsure there. Probably I'd go with `Single Sign On` and `Single Log Out`?